content/en/armory-enterprise/quick-spin-instructions/_index.md

```diff
@@ -4,28 +4,37 @@ linkTitle: "quick-spin-instructions"
 no_list: true
 weight: 3
 description: >
-  Get an evaluation or testing instance of Spinnaker up and running.
+  Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory's Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
 exclude_search: true
 toc_hide: true
 ---
-{{< include "experimental-feature.html" >}}
 
-## What is Quick Spin?
-Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product.
+> The Quick Spin container is not intended for production use.
 
-Once you have Quick Spin running, you can use it to evaluate Armory CD Self Hosted and to learn how to use Spinnaker. You can also use Quick Spin to test out Spinnaker features and to learn how to configure Spinnaker. 
+{{% heading "prereq" %}}
 
-Included with the Quick Spin container is a sample application and three sample pipelines that you can use to test out Spinnaker. After installing Quick Spin, you can use the sample application and pipelines to test out Spinnaker features.
+To run the Quick `quick-spin` container you need a host machine that can support a running instance with 
+  - 4 vCPU
+  - 8GB Memory
+  - 2GB Disk space
+  - The host machine must have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. Docker Compose is installed with Docker Desktop.
 
+## Installing Quick Spin
 
-> The Quick Spin container is not intended for production use.
+Open a terminal to the preferred installation directory and run the following docker command:
 
-## Installing Quick Spin
+```shell
+ docker compose pull quickspin armory/quickspin:latest
```

Comment:
The Docker image is `amory/quick-spin`
https://hub.docker.com/r/armory/quick-spin

Reply:
Also, the command is `docker pull armory/quick-spin`

Reply:
And then you have to run the container
`docker run --name quick-spin --rm -p 9000:80 -t armory/quick-spin:latest`

Reply:
the commands for pulling and running the image are on the dockerhub page

Reply:
After listing the commands to pull and run, how about adding the success message displayed in the terminal?

Something like

Upon success, you should see a message like:
```
```bash
 Waiting for Quick-Spin to become ready . . .
    +---------------------------------+
    |                                 |
    | Quick-Spin is ready to go!      |
    |                                 |
    | Go to: http://localhost:9000    |
    |                                 |
    | To stop Quick-Spin press ctrl+c |
    |                                 |
    +---------------------------------+
```
```